test/runtime/specs/juttle-spec/flowgraph-tests.spec.md

```diff
@@ -14,3 +14,21 @@ Modifying a point on one branch does not modify it on another.
     {"label":"branch1"}
     {"label":"branch1"}
     {"label":"pre"}
+
+Pints with `time == null` are handled correctly
+-----------------------------------------------
+
+Regression test for #351.
+
+### Juttle
+
+    emit -from :0: -limit 1
+    | batch -every :1s:
+    | filter true = false
+    | (reduce time = null; reduce time = null)
```

Comment:
to more directly match the situation I ran into, how about making time=last(time)? I know the result is the same either way.